scripts/fetch-snapshot.bash

```diff
@@ -20,9 +20,10 @@ fi
 target_dir_absolute="$(realpath $target_dir)"
 echo "Target path: $target_dir_absolute" >&2
 
+marinade_gs_bucket="gs://marinade-solana-snapshot-mainnet"
 jito_gs_bucket="gs://jito-mainnet"
 
-gs_files=$(gcloud storage ls "$jito_gs_bucket/$epoch/**/*.tar.zst" || exit 1)
+gs_files=$(gcloud storage ls "$marinade_gs_bucket/$epoch/**/*.tar.zst" || gcloud storage ls "$jito_gs_bucket/$epoch/**/*.tar.zst" || exit 1)
```

Comment:
Don't we need here as well the

```
gs_files=$(gcloud storage ls "$marinade_gs_bucket/$epoch/**/*.tar.zst" || gcloud storage ls "$jito_gs_bucket/$epoch/**/*.tar.zst" || exit 1) || exit 1
```

Reply:
You were right, the exit at then end is useless